src/cdk-experimental/combobox/combobox.ts

```diff
@@ -103,6 +112,45 @@ export class CdkCombobox<T = unknown> implements OnDestroy, AfterContentInit {
     this.panelValueChanged.complete();
   }
 
+  keydown(event: KeyboardEvent) {
```

Comment:
```suggestion
  _keydown(event: KeyboardEvent) {
```
Should have an underscore since this isn't a public API